norfair/lib_metrics.py

```diff
@@ -0,0 +1,336 @@
+import os
+import time
+import sys
+import numpy as np
+import random
+import cv2
+from rich import print
+from .utils import validate_points
+from rich.progress import BarColumn, Progress, TimeRemainingColumn, track
+from norfair import Detection
+import norfair
+import motmetrics as mm
+from collections import OrderedDict
+import pandas as pd
+
+
+def search_value_on_document(file_path, value_name):
```

Comment:
I do not think that there could be an increase in perfomance by changing this, because I just use this function at the beginning, just when I start to proccess each video. It is not a function that I will be using all along the whole proccess. But I will turn it into a class instead if you think that is easier or more intuitive to use that way.